content/blog/2019/01/2019-01-09-jenkins-user-conference-china-shenzhen-update.adoc

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "Jenkins User Conference China - Shenzhen Update"
+tags:
+- jenkins
+- jenkinsuserconference
+author: ijyun
+---
+
+
+On November 3rd, 2018 the Jenkins User Conference China (JUCC) met Jenkins users in Shenzhen which is the most burgeoning city in China. 
+It was the first time to hold JUCC in Shenzhen. 
+We held JUCC along with DevOps International Summit, which is the biggest DevOps event in China.
+More than 200 attendees gathered at JUCC Shenzhen to share and discuss Jenkins, DevOps, Continuous Delivery, Pipeline, and Agile. 
+
+
+Below, I am sharing pictures and some of the topics discussed at the event:
+
+image::/images/post-images/2019-01-09-jucc-images/image1.jpg[role="center",width=800]
+Yu Gu from Accenture presented **New challenges for DevOps in Cloud Native**.
+
+image::/images/post-images/2019-01-09-jucc-images/image2.jpg[role="center",width=800]
```

Comment:
@svanalstine This is a perfect example of why you want to use only lowercase in your file names: This file is named "image2.JPG" in the repo, but you're pointing to "image2.jpg" here. 

The reverse happens on some of your other images links.  😭 
I'll go ahead and clean these up and get this out.